bin/hdfcoinc/pycbc_page_segtable

```diff
@@ -108,8 +108,11 @@ for segment_name in opts.segment_names:
 
     # put comment in caption
     caption += " " + segment_name
-    if comment_dict[key] != None:
-        caption += " ("+comment_dict[key]+")"
```

Comment:
@cmbiwer is this your code? As far as I can see it never really worked (it didn't fail, but I'm not sure it would do anything meaningful in many cases). If you want to fix this, please do so on this development branch, or I'll just push to master in this form to be fixed, or removed, later.